ExchangeWebServices/Source/MS-OXWSCORE/TestSuite/MS-OXWSCORE_RequirementSpecification.xml

```diff
@@ -3941,7 +3957,7 @@
 	<ns1:Requirement>
 		<ns1:REQ_ID>MS-OXWSCORE_R1065</ns1:REQ_ID>

```

Comment:
It should be derived RS to verify like MS-OXWSCORE_R1730. Please add New RS and Comments.